modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/NumberUtil.java

```diff
@@ -39,4 +40,8 @@ public static boolean hasDecimalSeparator(String value) {
 		return true;
 	}
 
+	private static final String _DECIMAL_SEPARATORS = StringBundler.concat(
+		StringPool.PERIOD, StringPool.COMMA,
+		StringPool.ARABIC_DECIMAL_SEPARATOR);
```

Comment:
To apply my suggestion, this needs to be an array and please sort it alphabetically like so:
```suggestion
	private static final String[] _DECIMAL_SEPARATORS = {
		StringPool.ARABIC_DECIMAL_SEPARATOR, StringPool.COMMA, StringPool.PERIOD
	};
```